vortex-api/src/main/java/mil/army/usace/hec/vortex/io/VortexGridCollection.java

```diff
@@ -0,0 +1,169 @@
+package mil.army.usace.hec.vortex.io;
+
+import mil.army.usace.hec.vortex.VortexGrid;
+import mil.army.usace.hec.vortex.geo.WktParser;
+import mil.army.usace.hec.vortex.util.VortexTimeUtils;
+import ucar.nc2.constants.CF;
+import ucar.unidata.geoloc.Projection;
+import ucar.unidata.geoloc.projection.LatLonProjection;
+import ucar.unidata.util.Parameter;
+
+import java.time.Duration;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Predicate;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+class VortexGridCollection {
+    private static final Logger logger = Logger.getLogger(VortexGridCollection.class.getName());
+
+    private final List<VortexGrid> vortexGridList;
+    private final VortexGrid defaultGrid;
+    private final double[] xCoordinates;
+    private final double[] yCoordinates;
+    private final ZonedDateTime baseTime;
+
+    public VortexGridCollection(List<VortexGrid> vortexGrids) {
```

Comment:
Update `VortexGridCollection` public functions to package-private

Reply:
I think type safety would be better with List instead of Array -- [[Effective Java: Prefer lists to arrays]](https://www.informit.com/articles/article.aspx?p=2861454&seqNum=3)

Reply:
Added factory methods and make constructors that take in List<> as an arg private.